### ISSUE TITLE
Request: allow moving tabs across sessions

### ISSUE THREAD

**Issue by egabrum** [tim-we/tabs-aside#94]
I think Tabs Aside can be a OneTab killer... but there's a feature that I miss:

Allow moving tabs from one session to another. 
This would be done in the sidebar, I suppose.  Ultimately, a dedicated interface with other features (e.g. filtering) would be nice, but the sidebar would do for now.

Rationale: Sometimes sessions are not based on time, but on subject. In this case, one might need to move/group tabs from one session into another.

**Comment by tim-we** [comment]
I agree this would be useful feature. Unfortunately it also involves a lot of work and I do not have that much time right now. At some point in the future I will implement it but not right now, sorry.
